app/services/bulk_upload/sales/year2023/row_parser.rb

```diff
@@ -562,6 +563,15 @@ def validate_data_protection_answered
     end
   end
 
+  def validate_buyers_organisations
+    organisations_fields = %i[field_67 field_68 field_69 field_70]
+    if organisations_fields.all? { |field| attributes[field.to_s].blank? }
```

Comment:
The error message below suggests this should be any? — please could you confirm?